packages/fast-usdc/src/exos/advancer.js

```diff
@@ -251,9 +257,9 @@ export const prepareAdvancerKit = (
           }
         },
       },
-      transferHandler: {
+      transferOrSendHandler: {
```

Comment:
this distinction isn't helpful. I think It could still be `transferHandler` and consider `send` a kind of transfer. 

The `onFulfilled` method logs "Advance transfer fulfilled" and the verb should be consistent.

If you want "transfer" to imply "IBC transfer" then I think a new verb is should be defined that encompasses IBC transfer and Bank send. I would argue that "transfer" serves that already but would be open to better ideas.

Reply:
I considered parameterizing the log, but came to a similar conclusion that this is effectively a transfer.

The pattern of `method*Handler` from `@agoric/orchestration` for vow handlers was also a little sticky in my brain here - but maybe we should avoid it. I'm onboard with `transferHandler`, but maybe `advanceHandler` is more encompassing and distinct?  I can also reword the log to a simpler "Advance succeeded"